es-design-system/pages/molecules/es-form-msg.vue

```diff
@@ -8,17 +8,42 @@
                 bootstrap-vue alert
             </b-link>
         </p>
-        <es-button @click="fakeFormMsg()">
-            Show Success
+
+        <es-button
+            @click="showSuccess = true">
+            Show success
+        </es-button>
+        <es-button
+            @click="showError = true">
+            Show error
         </es-button>
-        <es-button @click="fakeFormMsg(false)">
-            Show Error
+        <es-button
+            @click="showInfo = true">
+            Show info
         </es-button>
+
+        <es-form-msg
+            class="my-450"
+            :show="showSuccess"
+            :variant="success"
```

Comment:
I'm getting console warnings and the EsFormMsg docs page examples are the wrong color. We'll need to change this to `variant="success"` because with the colon in front, it's looking for a variable named `success` and not finding it, so it's passing in `undefined` and EsFormMsg is defaulting to `danger` because that's the default value of the `variant` prop.